src/main/java/org/troyargonauts/subsystems/DriveTrain.java

```diff
@@ -132,26 +133,26 @@ public void resetEncoders() {
      * @return the angle to 0
      */
     
-//    public void resetAngle() {
-//        pigeon.setYaw(0);
-//    }
+    public void resetAngle() {
+        pigeon.setYaw(0);
+    }
 
     
     /** 
      * Returns angles between -180 and 180 degrees from pigeon
      * @return angle of robot
      */
-//    public double getAngle() {
-//        double output = pigeon.getYaw() % 360;
-//        while (Math.abs(output) > 180) {
-//            if (output < 0) {
-//                output += 360;
-//            } else {
-//                output -= 360;
-//            }
-//        }
-//        return output;
-//    }
+    public double getAngle() {
```

Comment:
Why are we only returning a value between -180 and 180? Robot rotates 360 degrees....